api/amendedJurisdictions/index.ts

```diff
@@ -4,25 +4,26 @@ import { config } from '../config'
 import { http } from '../lib/http'
 import { setHeaders } from '../lib/proxy'
 
+/**
+ * Manually filtering returned jurisdictions
+ * to make available jurisdiction in filters array only
+ */
 export async function getJurisdictions(req: express.Request, res: express.Response) {
+    console.log(req.url)
     let url = striptags(req.url)
+    console.log(url)
     url = req.baseUrl  + url
     const headers: any = setHeaders(req)
 
     try {
         const response = await http.get(`${config.services.ccd.componentApi}${url}`, { headers })
-
+        const filters = {jurisdiction: ['DIVORCE', 'PROBATE']}
```

Comment:
Is the extra `jurisdiction` layer necessary? `filters` could be an array instead I think.

Reply:
hmm dunno what thinking that it can be filtered by another thing in the future... what do u think?

Reply:
I think there won't be a need for that, but I see your point.